narayana-bom/pom.xml

```diff
@@ -24,7 +24,6 @@
         <groupId>org.apache.activemq</groupId>
         <artifactId>artemis-journal</artifactId>
         <version>${version.org.apache.activemq}</version>
-        <scope>provided</scope>
```

Comment:
If you remove the scope here then you need to check that the scope is present in the poms that use the dependency (such as https://github.com/jbosstm/narayana/blob/main/narayana-full/pom.xml#L155).

I think you need to check every usage to verify that the scope is right otherwise the dependency will use the default scope.

Reply:
Thanks Michael for your review. My idea is to have the default scope (compile) for every dependency without the scope set. In the case you mentioned (https://github.com/jbosstm/narayana/blob/main/narayana-full/pom.xml#L155) do you have any specific reason to set the scope as provided? 
I checked that dependency. The scope there has never been set. It had had compile scope until the bom was created and set it as provided.  With this PR the scope will be 'compile' again.

Reply:
The "previews scope" is what we had prior to pr/2079 ( [JBTM-3739] Create Narayana BOM #2079) so we'd need to check that PR in order to reinstate the original scope, unless the scope is still set in each module pom - perhaps @moulalis could provide us with some guidance.

Reply:
I confirm the scope is still set in each module pom (as it was before PR https://github.com/jbosstm/narayana/pull/2079), is it right @moulalis ?

Reply:
> I confirm the scope is still set in each module pom (as it was before PR #2079), is it right @moulalis ?

Yes @marcosgopen, I didn't change scope in any of the module so it is as it was before.

Reply:
@jmfinelli from my point of view I consider this conversation as resolved but I will not press the `Resolve conversation` button since it it requires both of us to "agree to commit" to it!